lib/swoosh/adapters/local.ex

```diff
@@ -22,6 +22,11 @@ defmodule Swoosh.Adapters.Local do
   @behaviour Swoosh.Adapter
 
   def deliver(%Swoosh.Email{} = email, _config) do
-    Swoosh.InMemoryMailbox.push(email)
+    id =
```

Comment:
I eye-balled the code, but I don't see any reason it would?  The `MailboxPreview` isn't coupled to the delivery in any way...